### ISSUE TITLE
Variable font SubFamily names have a weird behavior on Windows

### ISSUE THREAD

**Issue by eduairet** [googlefonts/fontmake#825]
I'm attempting to export a variable font from a designspace file using the following command
`fontmake -m MyDesignSpace.designspace -o variable`
The output works well on almost every place (Mac, adobe, html) but in windows something weird happens, every style name is Regular, I couldn't find an arg or command to fix this problem, so I want to know if it is a bug or is just my script.
Thanks

**Comment by khaledhosny** [comment]
This is probably because the STAT table fontmake generates is minimal and does not include instance names https://github.com/MicrosoftDocs/typography-issues/issues/808

**Comment by anthrotype** [comment]
yes, that's a long standing issue. The problem is we (fontTools) may not have all the information to build a more complete STAT. There are efforts to extend the DesignSpace format to include that metadata (https://github.com/fonttools/fonttools/pull/2436).
   
In the meantime, I suggest you see if you can use https://github.com/daltonmaag/statmake to build a complete STAT table.

**Comment by eduairet** [comment]
The statmake does the trick, thanks a lot!
